config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -232,6 +232,76 @@ periodics:
           cpu: 2
           memory: "2Gi"
 
+- name: ci-kubernetes-kubemark-100-gce-scheduler-highqps
```

Comment:
Let me clarify one thing very explicitly.

While I agree this test makes sense to have (though probably I would put that in experimental file), as it will allow us to see where we are, track regressions, etc.
we are NOT close to be able to apply it in production clusters (at least by default).

We are not able to keep up with 100pods/s throughput in some usecases, and the P&F in the current shape doesn't address many of the issues yet (both handling treating lists differently than other requests, as well as addressing long-running requests (watch in particular) are on the roadmap, but they aren't done).

So I would like to make it very explicit that it's not something that I will agree (as SIG Scalability TL) to changing defaults any time soon.

Reply:
(adding some context for anyone stumbling upon this thread: we discussed this offline and everybody's on the same page -- this job is just one of many scalability tests that needs to be done before QPS limits are removed on the client-side)